src/core/GameKeyHandler.ttslua

```diff
@@ -120,6 +120,8 @@ function getColorToDiscardFor(hoveredObject, playerColor)
   local discardForMatColor
   if inArea(pos, areaNearPlaymat) then
     return closestMatColor
+  elseif pos.y > 3 then -- discard to closest mat if card is in a hand
```

Comment:
If you don't mind, could you use the actual value of a handzone instead of hardcoding this?
```lua
Player[playerColor].getHandTransform().position.y
```
This should do the trick (You can use any `handColor` since they'll all be the same, so I'm just using the triggering player's).